internal/host/plugin/repository_host_catalog.go

```diff
@@ -328,21 +314,18 @@ func (r *Repository) UpdateCatalog(ctx context.Context, c *HostCatalog, version
 		db.StdRetryCnt,
 		db.ExpBackoff{},
 		func(_ db.Reader, w db.Writer) error {
-			msgs := make([]*oplog.Message, 0, 3)
-			ticket, err := w.GetTicket(newCatalog)
```

Comment:
I think we still want to track this using tickets since we are updating 2 different tables (the catalog table and the catalog secret table) with the aggregate being the catalog.

So this would mean adding a db.NewOplogMsg to the delete and the create(onconflict) calls and writing the oplog entries with the messages like existed before.

Reply:
Fixed. Resolved this while I was resolving the merge conflict so that it still uses the tickets both in create and update.